pros/serial/ports/exceptions.py

```diff
@@ -1,8 +1,15 @@
+import os
+
+
 class ConnectionRefusedException(IOError):
     def __init__(self, port_name: str, reason: Exception):
         self.__cause__ = reason
         self.port_name = port_name
 
     def __str__(self):
+        extra = ''
+        if os.name == 'posix':
+            extra = 'adding yourself to dialout group '
```

Comment:
as far as I know, this only helps with errno 13. I'm not sure what causes errno 16

Reply:
You're right, but I wanted ConnectionRefusedException to just be a bucket list of things that you should make sure.